core/src/main/java/bisq/core/api/CorePriceService.java

```diff
@@ -41,29 +42,42 @@ public CorePriceService(PriceFeedService priceFeedService) {
         this.priceFeedService = priceFeedService;
     }
 
-    public void getMarketPrice(String currencyCode, Consumer<Double> resultHandler) {
```

Comment:
removed the unnecessary complexity created by the resultHandler instead of just returning the value